src/pe/import.rs

```diff
@@ -227,11 +227,12 @@ impl<'a> SyntheticImportDirectoryEntry<'a> {
                 opts,
             ) {
                 debug!("Synthesizing lookup table imports for {} lib, with import lookup table rva: {:#x}", name, import_lookup_table_rva);
-                let import_lookup_table = SyntheticImportLookupTableEntry::parse::<T>(
+                let import_lookup_table = SyntheticImportLookupTableEntry::parse_with_opts::<T>(
```

Comment:
@ko1N since you implemented this feature, does this look correct to you?

Reply:
I don't know why not passing the opts along would hurt, and seems to resolve @2vg 's issue (though I confess I'm not sure what was the exact problem), so I assume this is ok :)

Reply:
Yes, the table entry will not be resolved properly when not passing the opts in case rva_resolve is disabled.

Since we left all regular `parse` functions untouched (as in the regular/"old" behavior) this was an oversight on my part. Luckily those changes could've only caused trouble when using mapped binary dumps. Either way, the change look totally fine to me.